### PR TITLE
Add placeholder to the elementNotFound exception

### PR DESCRIPTION
**Pull request by jordi12100**
It was unclear for me that you can find an element by placeholder.

### CODE REVIEW

**Comment by aik099** [comment]
Please double check that `placeholder` attribute even can be used with elements, that searched using https://github.com/minkphp/Mink/blob/master/src/Selector/NamedSelector.php#L22-L161 , because currently you're adding `|placeholder` for button finding as well, but they surely don't have placeholders.

File: src/Element/TraversableElement.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ public function pressButton($locator)
         $button = $this->findButton($locator);
 
         if (null === $button) {
-            throw $this->elementNotFound('button', 'id|name|title|alt|value', $locator);
+            throw $this->elementNotFound('button', 'id|name|title|alt|value|placeholder', $locator);
```
Comment (stof) [inline src/Element/TraversableElement.php]: this does not make sense. Buttons don't have a placeholder

Comment (aik099) [inline src/Element/TraversableElement.php]: I said the same in my comment.

**Comment by jordi12100** [comment]
@aik099 is right:
"The placeholder attribute works with the following input types: text, search, url, tel, email, and password".

I've updated the PR. Only added it to the fillField method.

**Comment by aik099** [comment]
Also maybe it's worth to send PR to documentation (https://github.com/minkphp/docs) as well if we're mentioning `id|name|label|value` in there currently.

**Comment by jordi12100** [comment]
Just checked. It is already mentioned there.
**field**
Searches for a form field by its id, name, label or placeholder.

**Comment by stof** [comment]
@jordi12100 could you rebase your PR to fix conflicts ?

**Comment by jordi12100** [comment]
@stof, done! And the checks passed :wink:

**Comment by stof** [comment]
thanks

**Pull request merged**